app/domain/errors.rb

```diff
@@ -338,6 +343,26 @@ module AuthnAzure
         code: "CONJ00054E"
       )
     end
+
+    module AuthnGce
+
+      InvalidAudience = ::Util::TrackableErrorClass.new(
+        msg:  "'audience' token claim {0} is invalid. The format should be " \
+              "'conjur/<account_name>/<host_id>'",
+        code: "CONJ00067E"
+      )
+
+      TokenClaimNotFoundOrEmpty = ::Util::TrackableErrorClass.new(
+        msg:  "Claim '{0-claim-name}' not found or empty in token. " \
+              "Verify you request the token using 'format=full'",
```

Comment:
How about Verify the token is requested using 'format=full'?
We should assume no human is involved.

Reply:
well, a human wrote the `curl` to get the token so they should also add `format=full`. 

@shulifink wdyt?

Reply:
Are we combining empty and not found for security reasons, or is the result of an unfound claim an empty token?

Reply:
it's not a security concern. it's just that in both cases the result will be the same.

Reply:
Is this an instruction that needs to be carried out? Then  "Make sure to request the token using 'format=full'",

If something failed because it wasn;t done correctly, and now you are giving them a hint of why it fails:
"Make sure you requested the token using 'format=full'","

Reply:
it's the second one, thanks

Reply:
btw, i remember @yserota telling me to prefer `verify that` over `make sure`. did this change?